### ISSUE TITLE
default log level (warning) is too noisy

### ISSUE THREAD

**Issue by abiosoft** [containerd/nerdctl#390]
It appears that the log level is fixed to `warning` in the code and the only option provided is `--debug` flag.
I personally still find it to be too noisy and would prefer to have it at `info`.

Is this something that is supported?

Thanks.

**Comment by AkihiroSuda** [comment]
What warning log is noisy?

**Comment by abiosoft** [comment]
My outputs get cluttered with something like this quite often, as well as when using compose.

```
$ nerdctl -n k8s.io images
WARN[0000] unparsable image name "k8s.gcr.io/pause@sha256:1ff6c18fbef2045af6b9c16bf034cc421a29027b800e4f9b68ae9b1cb3e9ae07"
```

I understand the warning messages and they can be very helpful for troubleshooting purposes but in most cases I could care less about them and would rather disable them.

Thanks.

**Comment by AkihiroSuda** [comment]
We should just fix the broken name parser, without decreasing log level

**Comment by fahedouch** [comment]
@AkihiroSuda  this should be fixed with https://github.com/containerd/nerdctl/commit/180053d643025df358cc807bffcb832fc4278abd

@abiosoft  plz retry with nerdctl from main branch

**Comment by abiosoft** [comment]
@fahedouch same behaviour and I can confirm it's same version as master via the commit hash.
```
$ sudo ./nerdctl version
Client:
 Version:       v0.11.2-25-g180053d
 Git commit:    180053d643025df358cc807bffcb832fc4278abd

Server:
 containerd:
  Version:      v1.5.5
  GitCommit:    72cec4be58a9eb6b2910f5d10f1c01ca47d231c0
```

**Comment by AkihiroSuda** [comment]
Fixed in https://github.com/containerd/nerdctl/commit/a9f65bf981df6e501c28fea09306d847f74f4d2c

Will be included in v0.13.0
